.github/workflows/publishDockerToGhcr.yml

```diff
@@ -0,0 +1,41 @@
+name: Create and publish a Docker image
+
+on:
+  release:
+    types: [published]
```

Comment:
Good point about the image being ready when we click the release button.  I decided to configure the pipeline to start building on the push of a tag rather than the creation of a draft release because I felt that whoever is clicking the release button would like to have the chance to review the image before clicking the button to create the release and then publish it.